src/lib/models/containers.ts

```
@@ -0,0 +1,254 @@
+}
+
+export abstract class Container {
+	protected _created: boolean = false;
+
+	constructor(
+		protected readonly logger: MaximLogger,
+		protected readonly _id: string,
+		protected readonly _type: "trace" | "span",
+		protected readonly _name?: string,
+		protected _parentId?: string,
+		markCreated: boolean = false,
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)